### PR TITLE
141 modules upgrade

### PR DESCRIPTION
**Pull request by mbrancato31**
added disabling functionalities
- at first only the first 3 modules are enabled, then you have to complete the previous module to continue
-the quiz module lets you continue if you have the grade to pass

centered the videos and files being displayed

the quiz shows the answers if you have the grade to pass (its a little basic but functional)

keeping count of the  students that completes the course

general improvements

### CODE REVIEW

**Comment by dmols** [comment]
I basically just added the enrollment api into the module complete api, so that when the user completes a module (the following only occurs in the first module) the user is enrolled in the course, and the course contains the user id as an enrolled student. 

I also made it so that disenrolling does not affect the number of total students in the course. That value should reflect how many students took the course since it was created.

File: frontend/toolkit-webapp/src/pages/CourseCreatorNewCourse.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ function NewCourse(props) {
 
             const data = await res.json()
             if (data.message === undefined) {
-                const res = await fetch(config.server_url + config.paths.fileUpload + "?token=" + token + "&courseID=" + data._id + "&imageName=" + image.name, {
+                const res = await fetch(config.server_url + config.paths.fileUpload + "?token=" + token + "&courseID=" + data._id + "&imageName=" + image.name + "&typeofload=courseimage", {
```
Comment (dmols) [inline frontend/toolkit-webapp/src/pages/CourseCreatorNewCourse.js]: Could this cause issues for now?

**Review by dmols — APPROVED** [review]
Looks good, just left a comment on one of the file changes

File: frontend/toolkit-webapp/src/pages/CourseCreatorNewCourse.js (hunk 1 above)
Comment (mbrancato31) [reply frontend/toolkit-webapp/src/pages/CourseCreatorNewCourse.js]: i forgot about it, let me delete it

**Review by collyntheshots — APPROVED** [review]
Everything looked good and nothing is broken

**Pull request merged**